classes/World.py

```diff
@@ -47,3 +54,28 @@ def add_event(self, event: Event) -> None:
         """
         insert_index = bisect.bisect([event.time for event in self.stack], event.time)
         self.stack.insert(insert_index, event)
+
+    def add_trip_to_flow(self, start: int, end: int) -> None:
+        """
+        Adds a trip from start to end for cluster flow
+        :param start: departure cluster
+        :param end: arrival cluster
+        """
+        self.cluster_flow[(start, end)] += 1
+
+    def get_cluster_flow(self) -> [(int, int, int)]:
+        """
+        Get all flows between cluster since last vehicle arrival
+        :return: list: tuple (start, end, flow) flow from departure cluster to arrival cluster
+        """
+        return [
+            (start_end[0], start_end[1], flow)
+            for start_end, flow in list(self.cluster_flow.items())
```

Comment:
change to `[(start, end, flow) for (start, end), flow in self.cluster_flow.items()]`